scripts/background.js

```diff
@@ -1318,7 +1308,7 @@ function openThatContext(temp,url,methodOfShowing){
         }
       });
     }
-  }else if(methodOfShowing=='window'){
+  }else if(methodOfShowing=='window'){ //If context is to ve showed in window
```

Comment:
there are some typos in the code in general. Please review the comments wording again.